SQL/Archive/24.0/2019-07-01_fix_project_in_session.sql

```diff
@@ -0,0 +1,5 @@
+-- Populate new session field with pre recorded candidate project
+UPDATE session s JOIN candidate c ON s.CandID=c.CandID SET ProjectID=c.RegistrationProjectID WHERE ProjectId IS NULL;
+ALTER TABLE `session` CHANGE `ProjectID` `ProjectID` int(10) unsigned NOT NULL;
```

Comment:
Should this also be making the RegistrationProjectID on the candidate not nullable?

Also update the release notes.

Reply:
@changes made and notes added in #7980